packages/prose-editor/ProseEditor.js

```diff
@@ -54,7 +54,7 @@ ProseEditor.Prototype = function() {
   };
 
   this._initialize = function(props) {
-    var configurator = this.props.configurator;
```

Comment:
Ja es kracht bei willReceiveProps weil da nextProps reinkommen. Drum is ned aufgfallen.